docs/source/users.rst

```diff
@@ -0,0 +1,36 @@
+User Management
+===============
+
+This section covers the customer owner, adding and removing users, email on-boarding and user preferences.
+
+Customer Owner
+##############
+
+When a customer is created a user is also created associated with the customer. This initial user is referred to a the customer owner and has special privileges. A customer owner has the following privileges:
+
+- Adding new users
+- Removing users
+- Deleting providers
+
+Adding and Removing Users
+#########################
+
+In order to allow more people to access to view the cost data, the customer owner can add more users to their customer. In order to add a user you must supply a username and valid email for the associated user. A password can be optionally provided, if a password is not provided one will be generated for the user. When the user is created they will receive an email to assist with on-boarding, see the section on `Email On-boarding`_ for more information.
```

Comment:
Extra words - `allow more people >>to access to view<< the cost data,`

Also, the more I read "customer owner" in the docs, the more I think these terms aren't ideal. I think we may need to reframe them into a more user-centered context. "Customer" just doesn't see correct in terms of what a user will identify with.